src/PlaywrightSharp.Tests/Browser/CloseTests.cs

```diff
@@ -9,6 +9,7 @@ namespace PlaywrightSharp.Tests.Browser
     ///<playwright-file>launcher.spec.js</playwright-file>
     ///<playwright-describe>Browser.close</playwright-describe>
     [Trait("Category", "chromium")]
+    [Trait("Category", "firefox")]
     [Collection(TestConstants.TestFixtureCollectionName)]
     public class CloseTests : PlaywrightSharpBaseTest
     {

```

Comment:
`ShouldTerminateNetworkWaiters` doesn't seem to be able to cancel `WaitForRequestAsync` & `WaitForResponseAsync`, a bit stuck here